src/analysis/multiSpecies/microbiomeModelingToolbox/pairwiseInteractionModeling/printUptakeBoundCom.m

```diff
@@ -11,7 +11,7 @@ function printUptakeBoundCom(model, SpFlag, metFlag)
 %
 % OPTIONAL INPUTS:
 %    SpFlag:   true to show individual uptake rates though community uptake is not allowed (default false)
```

Comment:
The output arguments need to be documented.

Reply:
@laurentheirendt, added.